template/public/index.html

```diff
@@ -24,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>React Redux App</title>
```

Comment:
Did we change the title of the JS one too?

Reply:
No

Reply:
yeah: https://github.com/reduxjs/cra-template-redux/blob/master/template/public/index.html#L27